src/js/structs/Node.js

```diff
@@ -73,7 +73,10 @@ class Node extends Item {
     let sum = 0;
 
     Object.keys(TaskStates).forEach(function(taskType) {
-      if (TaskStates[taskType].stateTypes.indexOf(state) !== -1) {
+      if (
+        TaskStates[taskType] &&
```

Comment:
I wonder what is the `taskType` we don't have mapping for? Could you timebox a research please?